modules/virt-creating-pvcs-to-store-default-os-images.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-and-using-default-os-//images.adoc
```

Comment:
A // in the file name here, too.

Reply:
garbage in... garbage out! Thanks. Fixed.